src/rgw/support/rgw_support_bundle

```diff
@@ -107,10 +106,18 @@ class RGWSupportBundle:
                 outfile = os.path.join(tmp_motr_trace_dir, file)
                 # Convert m0trace file to human readable yaml format
                 SimpleProcess(f'm0trace -i {infile} -Y -o {outfile}.yaml').run()
+                SimpleProcess(f'xz {outfile}.yaml').run() # compress the output file
 
         if stacktrace:
-            # TODO: CORTX-32151 Generate live stack trace for rgw process
-            pass
+            from subprocess import check_output
+            process = '/usr/bin/radosgw'
+            pid = check_output(['pidof', process]).decode('utf-8').split('\n')[0]
+            stacktrace_file = os.path.join(RGWSupportBundle._tmp_src, 'rgw_live_callstack.log')
+            cmd = """gdb --batch --quiet -ex "thread apply all bt full" """\
```

Comment:
Can you add some code comment here explaining this command.

Did you get this from Motr team?

Reply:
they are using GBD too, not in the exact same way.

Reply:
this is taken from legacy s3 server 
https://github.com/Seagate/cortx-s3server/blob/main/scripts/s3-support-bundles/s3_bundle_generate.sh#L234 

@papan-singh , @madhavemuri please verify if anything needs to be added in the stacktrace

Reply:
@papan-singh @madhavemuri Stack trace generation approach in PR above is kept similar to what was in legacy cotx-s3server, still please confirm that this is what is expected

Reply:
if gdb is not present on the system then we should not terminate the SB generation script

Reply:
@papan-singh  ,current behaviour is the same. if gdb is not present, support bundle will not break

Reply:
> Can you add some code comment here explaining this command.

i dont see anything for this yet!

Reply:
REMOVED `gbd` PACKAGE

[root@cortx-server-headless-svc-ssc-vm-rhev4-2782 /]# rpm -q gdb
gdb-8.2-18.el8.x86_64
[root@cortx-server-headless-svc-ssc-vm-rhev4-2782 /]# rpm -e --nodeps gdb-8.2-18.el8.x86_64
[root@cortx-server-headless-svc-ssc-vm-rhev4-2782 /]# rpm -q gdb
package gdb is not installed
[root@cortx-server-headless-svc-ssc-vm-rhev4-2782 /]#


TRIGGERED LOG COLLECTOR SCRIPT

```
[root@ssc-vm-rhev4-2782 k8_cortx_cloud]# ./logs-cortx-cloud.sh -s solution.yaml --all True
######################################################
# ✍️  Generating logs, namespace: cortx, date: 2022-06-27_04-54
######################################################

🔍 Logging pod: cortx-consul-client-785dt
🔍 Logging pod: cortx-consul-server-0
🔍 Logging pod: cortx-control-857cc458-dccbv, container: cortx-csm-agent
🔍 Logging pod: cortx-control-857cc458-dccbv, container: cortx-setup
 ⭐ Generating support-bundle logs for pod: cortx-control-857cc458-dccbv
Please use the below bundle id for checking the status of support bundle.
-------------------------------------------------------------
| bundle-logs-cortx-control-857cc458-dccbv-2022-06-27_04-54 |
-------------------------------------------------------------
Please Find the file on -> /var/cortx/support_bundle/bundle-logs-cortx-control-857cc458-dccbv-2022-06-27_04-54 .

tar: Removing leading `/' from member names

🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: cortx-hax
🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: cortx-motr-confd
🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: cortx-motr-io-001
🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: cortx-motr-io-002
🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: cortx-setup
🔍 Logging pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg, container: node-config
 ⭐ Generating support-bundle logs for pod: cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg
Please use the below bundle id for checking the status of support bundle.
------------------------------------------------------------------------------
| bundle-logs-cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg-2022-06-27_04-54 |
------------------------------------------------------------------------------
Please Find the file on -> /var/cortx/support_bundle/bundle-logs-cortx-data-ssc-vm-rhev4-2782-76447c7d85-7xhzg-2022-06-27_04-54 .

tar: Removing leading `/' from member names

🔍 Logging pod: cortx-ha-6fbc4d5c97-w6nlb, container: cortx-ha-fault-tolerance
🔍 Logging pod: cortx-ha-6fbc4d5c97-w6nlb, container: cortx-ha-health-monitor
🔍 Logging pod: cortx-ha-6fbc4d5c97-w6nlb, container: cortx-ha-k8s-monitor
🔍 Logging pod: cortx-ha-6fbc4d5c97-w6nlb, container: cortx-setup
 ⭐ Generating support-bundle logs for pod: cortx-ha-6fbc4d5c97-w6nlb
Please use the below bundle id for checking the status of support bundle.
----------------------------------------------------------
| bundle-logs-cortx-ha-6fbc4d5c97-w6nlb-2022-06-27_04-54 |
----------------------------------------------------------
Please Find the file on -> /var/cortx/support_bundle/bundle-logs-cortx-ha-6fbc4d5c97-w6nlb-2022-06-27_04-54 .

tar: Removing leading `/' from member names

🔍 Logging pod: cortx-kafka-0
🔍 Logging pod: cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q, container: cortx-rgw
🔍 Logging pod: cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q, container: cortx-hax
🔍 Logging pod: cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q, container: cortx-setup
 ⭐ Generating support-bundle logs for pod: cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q
Please use the below bundle id for checking the status of support bundle.
--------------------------------------------------------------------------------
| bundle-logs-cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q-2022-06-27_04-54 |
--------------------------------------------------------------------------------
Please Find the file on -> /var/cortx/support_bundle/bundle-logs-cortx-server-ssc-vm-rhev4-2782-65fbfd74b7-wg56q-2022-06-27_04-54 .

tar: Removing leading `/' from member names

🔍 Logging pod: cortx-zookeeper-0

📦 "logs-cortx-cloud-2022-06-27_04-54.tar" file generated
✔️  All done

[root@ssc-vm-rhev4-2782 k8_cortx_cloud]# 
```

STACKTRACE FILE 

```
[root@ssc-vm-rhev4-2782 rgw]# cat rgw_live_callstack.log
gbd package missing! skipping stacktrace generation[root@ssc-vm-rhev4-2782 rgw]#
```

Reply:
Script should also log about missing pkg

Reply:
done